app/routes/vaccinate.js

```diff
@@ -124,6 +126,66 @@ module.exports = router => {
     }
   })
 
+  router.get('/vaccinate/patient', (req, res) => {
+
+    const data = req.session.data
+    const showError = data.showError
+    const nhsNumberKnown = data.nhsNumberKnown
+    const nhsNumber = String(data.nhsNumber).replaceAll(' ', '')
+
+    let nhsNumberKnownError, nhsNumberError
+    let errorList = []
+
+    if (req.query.showError == 'yes') {
+
+      if (!nhsNumberKnown) {
+        nhsNumberKnownError = {
+          text: "Select if you have the patient’s NHS number",
```

Comment:
Update from @Anna-Sutton:
```suggestion
          text: "Select yes if you have the patient's NHS number",
```